docs/finalproject/prep.md

```diff
@@ -9,7 +9,7 @@ Getting the Final Projects ready is not a task that can be done a week before th
 
 **When you are on the React Module**
 - [ ] Must review the trainee’s performance and make [readiness for Final Projects decision](https://docs.google.com/document/d/1jMxqI0L7IKFENCQ8Lw-D1lhnj3c3RZI_WK808wG4YJM/edit?usp=sharing)
-- [ ] Identify trainees that will be filling in roles such as UI/UX Designer, Tester or Product Delivery
+- [ ] If you haven't yet, make sure you dentify trainees that will be filling in roles such as UI/UX Designer, Tester or Product Delivery
 - [ ] Ask internally and externally for possible ideas of charities that could be part of Final Projects. The form used should be [this](https://forms.gle/Gn85Kdna6QB2Qfcp9). 
 - [ ] Start recruiting Tech Lead, Product Managers and UI/UX Designer for Final Projects. See the role descriptions [here](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles). 
```

Comment:
```suggestion
- [ ] Start recruiting Tech Lead, Product Managers and UI/UX Designer for Final Projects. See the [role descriptions](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles). 
```

Links must make sense out of context. https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html